### PR TITLE
'Can't execute code from a freed script' error for document.write in iframe with same-domain src (close #384)

### LINKED ISSUE #384: 'Can't execute code from a freed script' error for document.write in iframe with same-domain src
Test for reproduce:
```javascript
   var iframe = document.createElement('iframe');
    var src    = window.QUnitGlobals.getResourceUrl('index.html');

    iframe.id = 'test_ojfnnhsg43';
    iframe.setAttribute('src', src);
    window.QUnitGlobals.waitForIframe(iframe)
        .then(function () {
            var iframeDocument = iframe.contentDocument;
            
            iframeDocument.write('<h1>test</h1>'); //this line throws the error
            
            iframe.parentNode.removeChild(iframe);
            start();
        });
    document.body.appendChild(iframe);
```

```javascript
index.html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title></title>
    <script src="/hammerhead.js" class="script-hammerhead-shadow-ui"></script>
</head>
<body>
    <script type="text/javascript">
        var hammerhead = window['%hammerhead%'];

        hammerhead.get('./utils/destination-location').forceLocation('http://localhost/sessionId/http://subdomain.example.com');
        hammerhead.start({
            sessionId: 'sessionId',
            CROSS_DOMAIN_PROXY_PORT: 2000
        });
    </script>
    <a href="/index.html">/index.html</a>
</body>
</html>
```

Only in IE and Edge.

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

File: src/client/sandbox/node/document/writer.js [hunk 1]
```diff
@@ -19,6 +20,31 @@ const REMOVE_CLOSING_TAG    = /<\/[^>]+>$/g;
 const PENDING_RE            = /<[A-Za-z][^>]*$/g;
 const STORED_WRITE_INFO     = 'hammerhead|stored-write-info';
 
+const ON_WINDOW_RECREATION_SCRIPT_TEMPLATE = `
+    <script class="${ SHADOW_UI_CLASSNAME.script }" type="text/javascript">
+        (function () {debugger;
```
Comment (churkin) [inline src/client/sandbox/node/document/writer.js]: `debugger` ???

Comment (LavrovArtem) [inline src/client/sandbox/node/document/writer.js]: sorry it's my inattention

**Comment by LavrovArtem** [comment]
FPR

**Comment by miherlosev** [comment]
lgtm

**Comment by LavrovArtem** [comment]
ping @churkin

**Comment by churkin** [comment]
lgtm

**Pull request merged**